src/classlibnative/inc/nlsinfo.h

```diff
@@ -101,10 +101,8 @@ class COMNlsInfo {
     //
     static FCDECL1(FC_BOOL_RET, nativeInitCultureData, CultureDataBaseObject *data);
     static FCDECL3(FC_BOOL_RET, nativeGetNumberFormatInfoValues, StringObject* localeNameUNSAFE, NumberFormatInfo* nfi, CLR_BOOL useUserOverride);
-#ifdef FEATURE_USE_LCID
```

Comment:
Also, I know from enabling serialization in CoreCLR recently that there are places where FEATURE_USE_LCID impacts serialization, e.g.
https://github.com/dotnet/coreclr/blob/c940bc816db970c4dcb3130f6e1a8f3a51d73328/src/mscorlib/src/System/Globalization/CompareInfo.cs#L263-L268
(that was just the first one I saw just now when doing a search, but I believe there are others).  When these implementations were ported to the Unix counterparts, those pieces were left out, e.g.
https://github.com/dotnet/coreclr/blob/c940bc816db970c4dcb3130f6e1a8f3a51d73328/src/mscorlib/corefx/System/Globalization/CompareInfo.cs#L111
If we're bringing back these FEATURE_USE_LCID pieces in code that was then ported elsewhere, we need to ensure that any relevant newly-brought-back pieces are also re-ported.